app/server/static/components/annotator_qanda.vue

```diff
@@ -0,0 +1,179 @@
+<template lang="pug">
+  div(@click="setSelectedRange")
+    span.text-sequence(
+      v-for="r in chunks"
+      v-bind:class="getChunkClass(r)"
+      v-bind:style="getChunkStyle(r)"
+    ) {{ textPart(r) }}
+      button.delete.is-small(v-if="r.label==1", v-on:click="removeAnswer(r)")
+</template>
+
+<script>
+export default {
+  props: {
+    text: {
+      type: String,
+      default: '',
+    },
+    entityPositions: {
+      type: Array, // [{'startOffset': 10, 'endOffset': 15, 'label_id': 1}]
+      default: () => [],
+    },
+  },
+
+  data: () => ({
+    startOffset: 0,
+    endOffset: 0,
+  }),
+
+  computed: {
+    sortedEntityPositions() {
+      /* eslint-disable vue/no-side-effects-in-computed-properties */
+      this.entityPositions = this.entityPositions.sort((a, b) => a.start_offset - b.start_offset);
+      return this.entityPositions;
+      /* eslint-enable vue/no-side-effects-in-computed-properties */
+    },
+
+    chunks() {
+      const res = [];
+      let left = 0;
+      let e;
+      for (let i = 0; i < this.sortedEntityPositions.length; i++) {
+        e = this.sortedEntityPositions[i];
+        e.label = 1;
+        e.end_offset = e.start_offset + e.response.length;
+
+        const l = this.makeEmptyChunk(left, e.start_offset);
```

Comment:
Should this variable be called `label` or some-such?

Reply:
I renamed it `emptyChunk` because that's what they are :)